tala_connector.py

```diff
@@ -646,8 +659,12 @@ def _handle_synchronize_projects(self, param):
         scan_id = param['scan_id']
         project_ids = param['project_ids']
         try:
-            project_ids_formatted = [int(x) for x in project_ids.split(",")]
-        except:
+            project_ids_formatted = []
+            for x in project_ids.split(","):
+                if int(x) < 0:
```

Comment:
Ditto.

Reply:
Reverted back as I do not get a proper document about project id validation.